spark-connector/build.gradle.kts

```diff
@@ -27,6 +28,10 @@ dependencies {
   implementation("org.apache.kyuubi:kyuubi-spark-connector-hive_$scalaVersion:$kyuubiVersion")
   implementation("org.apache.spark:spark-catalyst_$scalaVersion:$sparkVersion")
   implementation("org.apache.spark:spark-sql_$scalaVersion:$sparkVersion")
+  implementation("org.scala-lang.modules:scala-java8-compat_$scalaVersion:$scalaJava8CompatVersion")
```

Comment:
Does it work in jdk 11 or 17?

Reply:
yes, it's `a Java 8 compatibility kit for Scala 2.12 and 2.11` which could pass IT. but the main problem is it doesn't support scala 2.13 or newer version.